src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -520,12 +520,10 @@ public static CultureInfo CurrentUICulture
         public static CultureInfo? DefaultThreadCurrentCulture
         {
             get => s_DefaultThreadCurrentCulture;
-            set
-            {
+            set =>
```

Comment:
nit: I would use:

```cs
// If you add pre-conditions to this method, check to see if you also need to
// add them to Thread.CurrentCulture.set.
set => s_DefaultThreadCurrentCulture = value;
```

instead of:

```cs
set =>
    // If you add pre-conditions to this method, check to see if you also need to
    // add them to Thread.CurrentCulture.set.
    s_DefaultThreadCurrentCulture = value;
```